tests/models/layoutlmv3/test_modeling_layoutlmv3.py

```diff
@@ -378,6 +379,28 @@ def test_model_from_pretrained(self):
         model = LayoutLMv3Model.from_pretrained(model_name)
         self.assertIsNotNone(model)
 
+    def test_sdpa(self):
+        model = LayoutLMv3Model.from_pretrained(
+            "hf-tiny-model-private/tiny-random-LayoutLMv3Model", attn_implementation="eager"
+        )
+        model_sdpa = LayoutLMv3Model.from_pretrained(
+            "hf-tiny-model-private/tiny-random-LayoutLMv3Model", attn_implementation="sdpa"
+        )
+
+        model = model.eval()
+        model_sdpa = model_sdpa.eval()
+
+        tokenizer = AutoTokenizer.from_pretrained("hf-tiny-model-private/tiny-random-LayoutLMv3Model")
+        words = "I am in Paris and".split()
+        inputs = tokenizer(text=[words], boxes=[[(0, 0, 1, 1)] * len(words)], return_tensors="pt")
+
+        with torch.no_grad():
+            res_eager = model(**inputs)
+            res_sdpa = model_sdpa(**inputs)
+            self.assertTrue(
+                torch.allclose(res_eager.last_hidden_state, res_sdpa.last_hidden_state, atol=1e-5, rtol=1e-4)
+            )
```

Comment:
Thanks for adding a test, but we have a common test `test_eager_matches_sdpa_inference`, so no need for another one. But it's great to see this one pass